arch/x86/core/Kconfig.ia32

```diff
@@ -84,6 +84,26 @@ config X86_USERSPACE
 	  supporting user-level threads that are protected from each other and
 	  from crashing the kernel.
 
+config X86_PF_USE_TSS
+	bool
+	default n
```

Comment:
Default is always `n`, so there is no need to explicitly say so.

Reply:
updated, thanks

Reply:
just add one board based on qemu_x86_lakemont with this config enabled, please check if it's doable, thanks.

Reply:
Could you break out the board addition into another commit?

Reply:
met issues when try to git rebase origin/main, please skip this commit
I may need to re-submit, thx